include/mbedtls/config.h

```diff
@@ -1652,18 +1652,26 @@
 /**
  * \def MBEDTLS_USE_PSA_CRYPTO
  *
- * Make the X.509 and TLS library use PSA for cryptographic operations, see
- * #MBEDTLS_PSA_CRYPTO_C.
+ * Make the X.509 and TLS library use PSA for cryptographic operations, and
+ * enable new APIs for using keys handled by PSA Crypto.
  *
- * Note: this option is still in progress, the full X.509 and TLS modules are
+ * \note This option is still in progress, the full X.509 and TLS modules are
```

Comment:
Minor: Split the sentence for better readability. For example: "This option is still in progress, and parts of the X.509 and TLS modules aren't yet ported to PSA. However, those parts will still continue to work as usual, so enabling this option should not break backwards compatibility."